tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -26,6 +26,23 @@
 )
 from tardis.montecarlo.montecarlo_numba import njit_dict
 from numba.typed import List
+import tqdm.notebook as tq
+from tqdm import tqdm
+
+packet_pbar = tq.tqdm(
+    total=860000,
+    desc="Packets",
+    bar_format="{bar}{percentage:3.0f}% packets propagated",
+)
+iteration_pbar = tq.tqdm(
+    total=20,
```

Comment:
Retrieve number of iterations from the tardis config object.  This is a user specified parameter.